frontend/src/scenes/ContactsPage.tsx

```diff
@@ -721,14 +720,12 @@ export default function ContactsPage({ locale, routePath }: SceneProps) {
             (user as any).destinationCountry
           )}
         </Typography>
-        {false && (
-          <Link to={PATHS.RECENT_CALLS} style={{ textDecoration: 'none' }}>
-            <ActionLink variant="body1" role="button">
-              <HistoryIcon />
-              {STRINGS[locale].CONTACTS_RECENT_CALLS}
-            </ActionLink>
-          </Link>
-        )}
+        <Link to={PATHS.RECENT_CALLS} style={{ textDecoration: 'none' }}>
```

Comment:
only this `false` was removed. other changes in file seem to be from previously unlinted file.